stories/dem-info.stories.jsx

```diff
@@ -8,7 +8,7 @@ export default {
 
 export const PrimaryAllFieldsIncluded = {
   args: {
-    dob: '1990-10-20T00:00:00.000Z',
+    dob: '1990-10-20',
```

Comment:
The test data is correct, it's an ISO format date string.  If necessary please fix the code to work with the data in the story.